src/Colors/Colors.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import './Colors.css';
+
+import { NavLink, Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const Colors = (props) => {
+  if (props.colors) {
+    return (
+      <div style={{ textAlign: "center" }}>
+        <NavLink to='/' className='home-nav'>home</NavLink>
+        <section className='colors'>
+          {props.colors.sort((a, b) => b.hex - a.hex).map(color => {
```

Comment:
Hoping to figure out how to sort these by hue, but am going to make that an extension since hex codes don't sort very easily.